documentation/all-articles.html

```diff
@@ -57,7 +57,8 @@ <h1>Northcoders News 🗞</h1>
     <div>
       <h2>Components</h2>
       <ol>
-        <li><code>App</code> (grey) contains the entire app.</li>
+        <li><code>Home</code> (grey) contains the entire screen.</li>
+        <li><code>Header</code> (green) displays the title.</li>
```

Comment:
Not sure that you need to be doing this file???

Reply:
It's my planning and design mock-up file. I do the design in HTML & CSS first (mostly). Then I can reuse that code in my .jsx files with a bit of tweaking.